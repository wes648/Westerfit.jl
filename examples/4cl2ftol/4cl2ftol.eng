   1,   1,  -1,   1,   0,   0,     0.0646994837
   1,   1,   0,   1,   0,   0,     0.0602352555
   1,   1,   1,   0,   0,   0,     0.0735149515
   1,   2,  -2,   1,   0,   0,     0.0804867909
   1,   2,  -1,   1,   0,   0,     0.0804608677
   1,   2,   0,   2,   0,   0,     0.0685259064
   1,   2,   1,   2,   0,   0,     0.0723398115
   1,   2,   2,   0,   0,   0,     0.0917259549
   3,   0,   0,   0,   0,   0,     0.0068044600
   3,   1,  -1,   1,   0,   0,     0.0144379355
   3,   1,   0,   1,   0,   0,     0.0106259319
   3,   1,   1,   0,   0,   0,     0.0225837114
   3,   2,  -2,   1,   0,   0,     0.0338237225
   3,   2,  -1,   1,   0,   0,     0.0338236423
   3,   2,   0,   2,   0,   0,     0.0023443973
   3,   2,   1,   2,   0,   0,     0.0225580371
   3,   2,   2,   0,   0,   0,     0.0491545190
   3,   3,  -3,   1,   0,   0,     0.0713244735
   3,   3,  -2,   1,   0,   0,     0.0675105682
   3,   3,  -1,   3,   0,   0,     0.0491545189
   3,   3,   0,   3,   0,   0,     0.0156078822
   3,   3,   1,   2,   0,   0,     0.0592199168
   3,   3,   2,   2,   0,   0,     0.0636841452
   3,   3,   3,   0,   0,   0,     0.0724996138
   5,   1,  -1,   1,   0,   0,     0.1350525983
   5,   1,   0,   1,   0,   0,     0.0642458701
   5,   1,   1,   0,   0,   0,     0.1362855753
   5,   2,  -2,   1,   0,   0,     0.1066051264
   5,   2,  -1,   1,   0,   0,     0.0912723424
   5,   2,   0,   2,   0,   0,     0.0718861990
   5,   2,   1,   2,   0,   0,     0.0800072553
   5,   2,   2,   0,   0,   0,     0.1257893179
   5,   3,  -3,   1,   0,   0,     0.1342449840
   5,   3,  -2,   1,   0,   0,     0.1334433907
   5,   3,  -1,   3,   0,   0,     0.1205241849
   5,   3,   0,   3,   0,   0,     0.0680722936
   5,   3,   1,   2,   0,   0,     0.1281645185
   5,   3,   2,   2,   0,   0,     0.1296256466
   5,   3,   3,   0,   0,   0,     0.1346289674
   5,   4,  -4,   1,   0,   0,     0.1243506118
   5,   4,  -3,   1,   0,   0,     0.1213167463
   5,   4,  -2,   3,   0,   0,     0.0912724236
   5,   4,  -1,   3,   0,   0,     0.0800331786
   5,   4,   0,   4,   0,   0,     0.0597816413
   5,   4,   1,   4,   0,   0,     0.0730613399
   5,   4,   2,   2,   0,   0,     0.1066051265
   5,   4,   3,   2,   0,   0,     0.1160599533
   5,   4,   4,   0,   0,   0,     0.1293396629
   7,   2,  -2,   1,   0,   0,     0.1345511113
   7,   2,  -1,   1,   0,   0,     0.1345511112
   7,   2,   0,   2,   0,   0,     0.1344097075
   7,   2,   1,   2,   0,   0,     0.1344097076
   7,   2,   2,   0,   0,   0,     0.1350502693
   7,   3,  -3,   1,   0,   0,     0.1391348025
   7,   3,  -2,   1,   0,   0,     0.1383474307
   7,   3,  -1,   3,   0,   0,     0.1191914907
   7,   3,   0,   3,   0,   0,     0.1079631738
   7,   3,   1,   2,   0,   0,     0.1300469059
   7,   3,   2,   2,   0,   0,     0.1338646614
   7,   3,   3,   0,   0,   0,     0.1411838035
   7,   4,  -4,   1,   0,   0,     0.1191913930
   7,   4,  -3,   1,   0,   0,     0.1190500871
   7,   4,  -2,   3,   0,   0,     0.1078217701
   7,   4,  -1,   3,   0,   0,     0.1077921533
   7,   4,   0,   4,   0,   0,     0.1009056633
   7,   4,   1,   4,   0,   0,     0.1010470672
   7,   4,   2,   2,   0,   0,     0.1079335569
   7,   4,   3,   2,   0,   0,     0.1190499894
   7,   4,   4,   0,   0,   0,     0.1217379508
   7,   5,  -5,   1,   0,   0,     0.1338459077
   7,   5,  -4,   1,   0,   0,     0.1262105476
   7,   5,  -3,   3,   0,   0,     0.0959570648
   7,   5,  -2,   3,   0,   0,     0.0958156613
   7,   5,  -1,   5,   0,   0,     0.0875399008
   7,   5,   0,   5,   0,   0,     0.0873984978
   7,   5,   1,   4,   0,   0,     0.0919206974
   7,   5,   2,   4,   0,   0,     0.0920621007
   7,   5,   3,   2,   0,   0,     0.0996570653
   7,   5,   4,   2,   0,   0,     0.0997984689
   7,   5,   5,   0,   0,   0,     0.1346333947
   9,   3,  -3,   1,   0,   0,     0.0914951054
   9,   3,  -2,   1,   0,   0,     0.0871150287
   9,   3,  -1,   3,   0,   0,     0.0549389637
   9,   3,   0,   3,   0,   0,     0.0395896281
   9,   3,   1,   2,   0,   0,     0.0549610221
   9,   3,   2,   2,   0,   0,     0.0869729067
   9,   3,   3,   0,   0,   0,     0.0916372278
   9,   4,  -4,   1,   0,   0,     0.0955321909
   9,   4,  -3,   1,   0,   0,     0.0953900682
   9,   4,  -2,   3,   0,   0,     0.0395676752
   9,   4,  -1,   3,   0,   0,     0.0395675697
   9,   4,   0,   4,   0,   0,     0.0283441617
   9,   4,   1,   4,   0,   0,     0.0283662201
   9,   4,   2,   2,   0,   0,     0.0395897336
   9,   4,   3,   2,   0,   0,     0.0549389636
   9,   4,   4,   0,   0,   0,     0.0992314718
   9,   5,  -5,   1,   0,   0,     0.1187665187
   9,   5,  -4,   1,   0,   0,     0.1186244934
   9,   5,  -3,   3,   0,   0,     0.1075086825
   9,   5,  -2,   3,   0,   0,     0.0283348965
   9,   5,  -1,   5,   0,   0,     0.0201934112
   9,   5,   0,   5,   0,   0,     0.0201713528
   9,   5,   1,   4,   0,   0,     0.0214518453
   9,   5,   2,   4,   0,   0,     0.0214739037
   9,   5,   3,   2,   0,   0,     0.1075382993
   9,   5,   4,   2,   0,   0,     0.1186243958
   9,   5,   5,   0,   0,   0,     0.1187666163
   9,   6,  -6,   1,   0,   0,     0.1073665595
   9,   6,  -5,   1,   0,   0,     0.1006221920
   9,   6,  -4,   3,   0,   0,     0.0549610220
   9,   6,  -3,   3,   0,   0,     0.0283128381
   9,   6,  -2,   5,   0,   0,     0.0124158287
   9,   6,  -1,   5,   0,   0,     0.0123937703
   9,   6,   0,   6,   0,   0,     0.0078469187
   9,   6,   1,   6,   0,   0,     0.0078689770
   9,   6,   2,   4,   0,   0,     0.0163176804
   9,   6,   3,   4,   0,   0,     0.0163397388
   9,   6,   4,   2,   0,   0,     0.0993735947
   9,   6,   5,   2,   0,   0,     0.1004800687
   9,   6,   6,   0,   0,   0,     0.1073961763
  11,   4,  -4,   1,   0,   0,     0.0913805219
  11,   4,  -3,   1,   0,   0,     0.0545018613
  11,   4,  -2,   3,   0,   0,     0.0119566683
  11,   4,  -1,   3,   0,   0,     0.0074319627
  11,   4,   0,   4,   0,   0,    -0.0569706909
  11,   4,   1,   4,   0,   0,     0.0074098169
  11,   4,   2,   2,   0,   0,     0.0119788141
  11,   4,   3,   2,   0,   0,     0.0391526132
  11,   4,   4,   0,   0,   0,     0.0915225473
  11,   5,  -5,   1,   0,   0,     0.0210147427
  11,   5,  -4,   1,   0,   0,     0.0197563963
  11,   5,  -3,   3,   0,   0,     0.0158805782
  11,   5,  -2,   3,   0,   0,    -0.0723524360
  11,   5,  -1,   5,   0,   0,    -0.0835702527
  11,   5,   0,   5,   0,   0,    -0.0835735054
  11,   5,   1,   4,   0,   0,    -0.0723558041
  11,   5,   2,   4,   0,   0,    -0.0723556887
  11,   5,   3,   2,   0,   0,     0.0159027240
  11,   5,   4,   2,   0,   0,     0.0197342505
  11,   5,   5,   0,   0,   0,     0.0210368885
  11,   6,  -6,   1,   0,   0,     0.0391304674
  11,   6,  -5,   1,   0,   0,     0.0279292052
  11,   6,  -4,   3,   0,   0,     0.0278757357
  11,   6,  -3,   3,   0,   0,    -0.0569674383
  11,   6,  -2,   5,   0,   0,    -0.0904349119
  11,   6,  -1,   5,   0,   0,    -0.0904381646
  11,   6,   0,   6,   0,   0,    -0.0917561595
  11,   6,   1,   6,   0,   0,    -0.0917529068
  11,   6,   2,   4,   0,   0,    -0.0836069187
  11,   6,   3,   4,   0,   0,    -0.0836036660
  11,   6,   4,   2,   0,   0,     0.0278978815
  11,   6,   5,   2,   0,   0,     0.0279070594
  11,   6,   6,   0,   0,   0,     0.0391305729
  11,   7,  -7,   1,   0,   0,     0.0868583204
  11,   7,  -6,   1,   0,   0,     0.0545240071
  11,   7,  -5,   3,   0,   0,     0.0391527187
  11,   7,  -4,   3,   0,   0,    -0.0569674382
  11,   7,  -3,   5,   0,   0,    -0.0956212393
  11,   7,  -2,   5,   0,   0,    -0.0956244920
  11,   7,  -1,   7,   0,   0,    -0.1041540689
  11,   7,   0,   7,   0,   0,    -0.1041573216
  11,   7,   1,   6,   0,   0,    -0.0995818556
  11,   7,   2,   6,   0,   0,    -0.0995786029
  11,   7,   3,   4,   0,   0,    -0.0723525514
  11,   7,   4,   4,   0,   0,    -0.0569706910
  11,   7,   5,   2,   0,   0,     0.0545018612
  11,   7,   6,   2,   0,   0,     0.0545240070
  11,   7,   7,   0,   0,   0,     0.0870003457
  13,   5,  -5,   1,   0,   0,     0.0118561991
  13,   5,  -4,   1,   0,   0,     0.0073315001
  13,   5,  -3,   3,   0,   0,    -0.1000283684
  13,   5,  -2,   3,   0,   0,    -0.1046005705
  13,   5,  -1,   5,   0,   0,    -0.2012661801
  13,   5,   0,   5,   0,   0,    -0.2012661802
  13,   5,   1,   4,   0,   0,    -0.2012657227
  13,   5,   2,   4,   0,   0,    -0.1046038343
  13,   5,   3,   2,   0,   0,    -0.1000251046
  13,   5,   4,   2,   0,   0,     0.0073093468
  13,   5,   5,   0,   0,   0,     0.0118783524
  13,   6,  -6,   1,   0,   0,    -0.0908814138
  13,   6,  -5,   1,   0,   0,    -0.0908846777
  13,   6,  -4,   3,   0,   0,    -0.0960677410
  13,   6,  -3,   3,   0,   0,    -0.0960710048
  13,   6,  -2,   5,   0,   0,    -0.2166669722
  13,   6,  -1,   5,   0,   0,    -0.2166670999
  13,   6,   0,   6,   0,   0,    -0.2278781529
  13,   6,   1,   6,   0,   0,    -0.2278776955
  13,   6,   2,   4,   0,   0,    -0.2166666425
  13,   6,   3,   4,   0,   0,    -0.2166665148
  13,   6,   4,   2,   0,   0,    -0.0922026724
  13,   6,   5,   2,   0,   0,    -0.0921994086
  13,   6,   6,   0,   0,   0,     0.0157801100
  13,   7,  -7,   1,   0,   0,     0.0158022633
  13,   7,  -6,   1,   0,   0,    -0.0728022017
  13,   7,  -5,   3,   0,   0,    -0.0840200183
  13,   7,  -4,   3,   0,   0,    -0.0840501679
  13,   7,  -3,   5,   0,   0,    -0.2279136320
  13,   7,  -2,   5,   0,   0,    -0.2279140894
  13,   7,  -1,   7,   0,   0,    -0.2360721229
  13,   7,   0,   7,   0,   0,    -0.2360725802
  13,   7,   1,   6,   0,   0,    -0.2347113559
  13,   7,   2,   6,   0,   0,    -0.2347108986
  13,   7,   3,   4,   0,   0,    -0.2012657229
  13,   7,   4,   4,   0,   0,    -0.0840534317
  13,   7,   5,   2,   0,   0,    -0.0840167545
  13,   7,   6,   2,   0,   0,    -0.0728023171
  13,   7,   7,   0,   0,   0,     0.0209142761
  13,   8,  -8,   1,   0,   0,     0.0277752682
  13,   8,  -7,   1,   0,   0,     0.0209364294
  13,   8,  -6,   3,   0,   0,    -0.0574139400
  13,   8,  -5,   3,   0,   0,    -0.0574139401
  13,   8,  -4,   5,   0,   0,    -0.0727989378
  13,   8,  -3,   5,   0,   0,    -0.0727990532
  13,   8,  -2,   7,   0,   0,    -0.2439528403
  13,   8,  -1,   7,   0,   0,    -0.2439532976
  13,   8,   0,   8,   0,   0,    -0.2485612768
  13,   8,   1,   8,   0,   0,    -0.2485608195
  13,   8,   2,   6,   0,   0,    -0.2399581073
  13,   8,   3,   6,   0,   0,    -0.2399576499
  13,   8,   4,   4,   0,   0,    -0.0574172040
  13,   8,   5,   4,   0,   0,    -0.0574172039
  13,   8,   6,   2,   0,   0,     0.0196337828
  13,   8,   7,   2,   0,   0,     0.0196559360
  13,   8,   8,   0,   0,   0,     0.0277974215
   1,   1,  -1,   1,   1,   1,     0.0646909926
   1,   1,   0,   1,   1,   1,     0.0602354839
   1,   1,   1,   0,   1,   1,     0.0750417341
   1,   2,  -2,   1,   1,   1,     0.0838281516
   1,   2,  -1,   1,   1,   1,     0.0775342012
   1,   2,   0,   2,   1,   1,     0.0686443681
   1,   2,   1,   2,   1,   1,     0.0715380493
   1,   2,   2,   0,   1,   1,     0.0876036329
   3,   0,   0,   0,   1,   1,     0.0067959032
   3,   1,  -1,   1,   1,   1,     0.0136331685
   3,   1,   0,   1,   1,   1,     0.0107450915
   3,   1,   1,   0,   1,   1,     0.0259255579
   3,   2,  -2,   1,   1,   1,     0.0383232288
   3,   2,  -1,   1,   1,   1,     0.0297011027
   3,   2,   0,   2,   1,   1,     0.0023446277
   3,   2,   1,   2,   1,   1,     0.0196306435
   3,   2,   2,   0,   1,   1,     0.0547297654
   3,   3,  -3,   1,   1,   1,     0.0705227114
   3,   3,  -2,   1,   1,   1,     0.0676290299
   3,   3,  -1,   3,   1,   1,     0.0439516643
   3,   3,   0,   3,   1,   1,     0.0171373545
   3,   3,   1,   2,   1,   1,     0.0592201452
   3,   3,   2,   2,   1,   1,     0.0636756541
   3,   3,   3,   0,   1,   1,     0.0740263962
   5,   1,  -1,   1,   1,   1,     0.1350528094
   5,   1,   0,   1,   1,   1,     0.0642373790
   5,   1,   1,   0,   1,   1,     0.1361503620
   5,   2,  -2,   1,   1,   1,     0.1014024862
   5,   2,  -1,   1,   1,   1,     0.0833745392
   5,   2,   0,   2,   1,   1,     0.0710844373
   5,   2,   1,   2,   1,   1,     0.0770805889
   5,   2,   2,   0,   1,   1,     0.1257809570
   5,   3,  -3,   1,   1,   1,     0.1333589095
   5,   3,  -2,   1,   1,   1,     0.1326476347
   5,   3,  -1,   3,   1,   1,     0.1244690730
   5,   3,   0,   3,   1,   1,     0.0681907552
   5,   3,   1,   2,   1,   1,     0.1273627588
   5,   3,   2,   2,   1,   1,     0.1308664432
   5,   3,   3,   0,   1,   1,     0.1342451951
   5,   4,  -4,   1,   1,   1,     0.1213169708
   5,   4,  -3,   1,   1,   1,     0.1205156939
   5,   4,  -2,   3,   1,   1,     0.0957717636
   5,   4,  -1,   3,   1,   1,     0.0871500206
   5,   4,   0,   4,   1,   1,     0.0597818697
   5,   4,   1,   4,   1,   1,     0.0745881221
   5,   4,   2,   2,   1,   1,     0.1121801850
   5,   4,   3,   2,   1,   1,     0.1160601817
   5,   4,   4,   0,   1,   1,     0.1297427205
   7,   2,  -2,   1,   1,   1,     0.1365716477
   7,   2,  -1,   1,   1,   1,     0.1346336058
   7,   2,   0,   2,   1,   1,     0.1330689234
   7,   2,   1,   2,   1,   1,     0.1338461188
   7,   2,   2,   0,   1,   1,     0.1390624614
   7,   3,  -3,   1,   1,   1,     0.1399821091
   7,   3,  -2,   1,   1,   1,     0.1391268769
   7,   3,  -1,   3,   1,   1,     0.1236884615
   7,   3,   0,   3,   1,   1,     0.1217381752
   7,   3,   1,   2,   1,   1,     0.1293514988
   7,   3,   2,   2,   1,   1,     0.1301639757
   7,   3,   3,   0,   1,   1,     0.1401235128
   7,   4,  -4,   1,   1,   1,     0.1150721715
   7,   4,  -3,   1,   1,   1,     0.1149307679
   7,   4,  -2,   3,   1,   1,     0.1050173774
   7,   4,  -1,   3,   1,   1,     0.1048759738
   7,   4,   0,   4,   1,   1,     0.0988966777
   7,   4,   1,   4,   1,   1,     0.1025364107
   7,   4,   2,   2,   1,   1,     0.1111561396
   7,   4,   3,   2,   1,   1,     0.1112975433
   7,   4,   4,   0,   1,   1,     0.1235470579
   7,   5,  -5,   1,   1,   1,     0.1292100951
   7,   5,  -4,   1,   1,   1,     0.1262021870
   7,   5,  -3,   3,   1,   1,     0.0960661535
   7,   5,  -2,   3,   1,   1,     0.0959247500
   7,   5,  -1,   5,   1,   1,     0.0875401028
   7,   5,   0,   5,   1,   1,     0.0873986998
   7,   5,   1,   4,   1,   1,     0.0919130721
   7,   5,   2,   4,   1,   1,     0.0920544753
   7,   5,   3,   2,   1,   1,     0.0990380814
   7,   5,   4,   2,   1,   1,     0.1023950070
   7,   5,   5,   0,   1,   1,     0.1383395067
   9,   3,  -3,   1,   1,   1,     0.0914874801
   9,   3,  -2,   1,   1,   1,     0.0871152307
   9,   3,  -1,   3,   1,   1,     0.0497627227
   9,   3,   0,   3,   1,   1,     0.0497406643
   9,   3,   1,   2,   1,   1,     0.0605322709
   9,   3,   2,   2,   1,   1,     0.0869731087
   9,   3,   3,   0,   1,   1,     0.0916296024
   9,   4,  -4,   1,   1,   1,     0.0956412797
   9,   4,  -3,   1,   1,   1,     0.0954991571
   9,   4,  -2,   3,   1,   1,     0.0354717657
   9,   4,  -1,   3,   1,   1,     0.0354497073
   9,   4,   0,   4,   1,   1,     0.0316754234
   9,   4,   1,   4,   1,   1,     0.0316974818
   9,   4,   2,   2,   1,   1,     0.0440636077
   9,   4,   3,   2,   1,   1,     0.0440856661
   9,   4,   4,   0,   1,   1,     0.0984710835
   9,   5,  -5,   1,   1,   1,     0.1231214643
   9,   5,  -4,   1,   1,   1,     0.1146472971
   9,   5,  -3,   3,   1,   1,     0.1107305460
   9,   5,  -2,   3,   1,   1,     0.0254233928
   9,   5,  -1,   5,   1,   1,     0.0194503503
   9,   5,   0,   5,   1,   1,     0.0194282919
   9,   5,   1,   4,   1,   1,     0.0229252789
   9,   5,   2,   4,   1,   1,     0.0229473374
   9,   5,   3,   2,   1,   1,     0.1108726689
   9,   5,   4,   2,   1,   1,     0.1145051742
   9,   5,   5,   0,   1,   1,     0.1232635872
   9,   6,  -6,   1,   1,   1,     0.1044503798
   9,   6,  -5,   1,   1,   1,     0.1021115360
   9,   6,  -4,   3,   1,   1,     0.0605102125
   9,   6,  -3,   3,   1,   1,     0.0254013344
   9,   6,  -2,   5,   1,   1,     0.0124085451
   9,   6,  -1,   5,   1,   1,     0.0123864867
   9,   6,   0,   6,   1,   1,     0.0078471104
   9,   6,   1,   6,   1,   1,     0.0078691688
   9,   6,   2,   4,   1,   1,     0.0164229718
   9,   6,   3,   4,   1,   1,     0.0164450302
   9,   6,   4,   2,   1,   1,     0.0986132066
   9,   6,   5,   2,   1,   1,     0.1019694129
   9,   6,   6,   0,   1,   1,     0.1045925029
  11,   4,  -4,   1,   1,   1,     0.0913728966
  11,   4,  -3,   1,   1,   1,     0.0493257076
  11,   4,  -2,   3,   1,   1,     0.0119493848
  11,   4,  -1,   3,   1,   1,     0.0074321544
  11,   4,   0,   4,   1,   1,    -0.0621641944
  11,   4,   1,   4,   1,   1,     0.0074100087
  11,   4,   2,   2,   1,   1,     0.0119715305
  11,   4,   3,   2,   1,   1,     0.0436265054
  11,   4,   4,   0,   1,   1,     0.0915149219
  11,   5,  -5,   1,   1,   1,     0.0224881765
  11,   5,  -4,   1,   1,   1,     0.0190133353
  11,   5,  -3,   3,   1,   1,     0.0159858696
  11,   5,  -2,   3,   1,   1,    -0.0678577325
  11,   5,  -1,   5,   1,   1,    -0.0802427039
  11,   5,   0,   5,   1,   1,    -0.0802459567
  11,   5,   1,   4,   1,   1,    -0.0764720578
  11,   5,   2,   4,   1,   1,    -0.0764688051
  11,   5,   3,   2,   1,   1,     0.0160080154
  11,   5,   4,   2,   1,   1,     0.0189911895
  11,   5,   5,   0,   1,   1,     0.0225103223
  11,   6,  -6,   1,   1,   1,     0.0350126049
  11,   6,  -5,   1,   1,   1,     0.0312604669
  11,   6,  -4,   3,   1,   1,     0.0249642320
  11,   6,  -3,   3,   1,   1,    -0.0514007975
  11,   6,  -2,   5,   1,   1,    -0.0889799247
  11,   6,  -1,   5,   1,   1,    -0.0889831774
  11,   6,   0,   6,   1,   1,    -0.0924793232
  11,   6,   1,   6,   1,   1,    -0.0924760705
  11,   6,   2,   4,   1,   1,    -0.0865128248
  11,   6,   3,   4,   1,   1,    -0.0865095721
  11,   6,   4,   2,   1,   1,     0.0249863778
  11,   6,   5,   2,   1,   1,     0.0312383211
  11,   6,   6,   0,   1,   1,     0.0350347507
  11,   7,  -7,   1,   1,   1,     0.0868585224
  11,   7,  -6,   1,   1,   1,     0.0600952558
  11,   7,  -5,   3,   1,   1,     0.0436486512
  11,   7,  -4,   3,   1,   1,    -0.0513975448
  11,   7,  -3,   5,   1,   1,    -0.0955202178
  11,   7,  -2,   5,   1,   1,    -0.0955234705
  11,   7,  -1,   7,   1,   1,    -0.1041538884
  11,   7,   0,   7,   1,   1,    -0.1041571412
  11,   7,   1,   6,   1,   1,    -0.0995887610
  11,   7,   2,   6,   1,   1,    -0.0995855083
  11,   7,   3,   4,   1,   1,    -0.0678609852
  11,   7,   4,   4,   1,   1,    -0.0621674471
  11,   7,   5,   2,   1,   1,     0.0493035618
  11,   7,   6,   2,   1,   1,     0.0600731100
  11,   7,   7,   0,   1,   1,     0.0870005477
  13,   5,  -5,   1,   1,   1,     0.0118489156
  13,   5,  -4,   1,   1,   1,     0.0073316919
  13,   5,  -3,   3,   1,   1,    -0.1000352738
  13,   5,  -2,   3,   1,   1,    -0.1046003900
  13,   5,  -1,   5,   1,   1,    -0.2064607003
  13,   5,   0,   5,   1,   1,    -0.2064611577
  13,   5,   1,   4,   1,   1,    -0.1956973920
  13,   5,   2,   4,   1,   1,    -0.1046036538
  13,   5,   3,   2,   1,   1,    -0.1000320100
  13,   5,   4,   2,   1,   1,     0.0073095386
  13,   5,   5,   0,   1,   1,     0.0118710688
  13,   6,  -6,   1,   1,   1,    -0.0894264266
  13,   6,  -5,   1,   1,   1,    -0.0894296904
  13,   6,  -4,   3,   1,   1,    -0.0959667195
  13,   6,  -3,   3,   1,   1,    -0.0959699834
  13,   6,  -2,   5,   1,   1,    -0.2207810268
  13,   6,  -1,   5,   1,   1,    -0.2207814842
  13,   6,   0,   6,   1,   1,    -0.2245549657
  13,   6,   1,   6,   1,   1,    -0.2245545083
  13,   6,   2,   4,   1,   1,    -0.2121736966
  13,   6,   3,   4,   1,   1,    -0.2121732392
  13,   6,   4,   2,   1,   1,    -0.0929258362
  13,   6,   5,   2,   1,   1,    -0.0929225723
  13,   6,   6,   0,   1,   1,     0.0158854013
  13,   7,  -7,   1,   1,   1,     0.0159075546
  13,   7,  -6,   1,   1,   1,    -0.0769153069
  13,   7,  -5,   3,   1,   1,    -0.0806924696
  13,   7,  -4,   3,   1,   1,    -0.0869560739
  13,   7,  -3,   5,   1,   1,    -0.2308129469
  13,   7,  -2,   5,   1,   1,    -0.2308134043
  13,   7,  -1,   7,   1,   1,    -0.2367729718
  13,   7,   0,   7,   1,   1,    -0.2367734292
  13,   7,   1,   6,   1,   1,    -0.2332772896
  13,   7,   2,   6,   1,   1,    -0.2332768322
  13,   7,   3,   4,   1,   1,    -0.1956978494
  13,   7,   4,   4,   1,   1,    -0.0869593377
  13,   7,   5,   2,   1,   1,    -0.0806892058
  13,   7,   6,   2,   1,   1,    -0.0769185707
  13,   7,   7,   0,   1,   1,     0.0223877094
  13,   8,  -8,   1,   1,   1,     0.0248637647
  13,   8,  -7,   1,   1,   1,     0.0224098626
  13,   8,  -6,   3,   1,   1,    -0.0518440466
  13,   8,  -5,   3,   1,   1,    -0.0518473105
  13,   8,  -4,   5,   1,   1,    -0.0683042343
  13,   8,  -3,   5,   1,   1,    -0.0683074981
  13,   8,  -2,   7,   1,   1,    -0.2439593389
  13,   8,  -1,   7,   1,   1,    -0.2439597963
  13,   8,   0,   8,   1,   1,    -0.2485611084
  13,   8,   1,   8,   1,   1,    -0.2485606510
  13,   8,   2,   6,   1,   1,    -0.2398617615
  13,   8,   3,   6,   1,   1,    -0.2398613042
  13,   8,   4,   4,   1,   1,    -0.0626139601
  13,   8,   5,   4,   1,   1,    -0.0626106962
  13,   8,   6,   2,   1,   1,     0.0188907223
  13,   8,   7,   2,   1,   1,     0.0189128756
  13,   8,   8,   0,   1,   1,     0.0248859179
