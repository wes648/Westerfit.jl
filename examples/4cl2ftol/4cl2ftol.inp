molnam
%CNTRLS
NFOLD = 3
S = 1.5
TK = 25.0
RUNmode = ESF
mcalc = 6
vtmax = 0
turducken = 1
Jmax = 6.5
νmin = 26.5
νmax = 40.0

%2NDORDER
A;              3036.286000;   1.0
B;               859.019910;   1.0
C;               672.365965;   1.0
Dab;            -124.023834;   1.0
ϵzz;                    0.0;   0.0
ϵxx;                    0.0;   0.0
ϵyy;                    0.0;   0.0
ϵxz;                    0.0;   0.0
χzz;            -69.8366520;   1.0
χxmy;            -2.9194263;   1.0
χxz;              .00000459;   1.0
F;               5.26805401;   0.5
ρ;      0.01843000000000000;   1.0
Vn;                  227.22;   1.0
η;                      0.0;   0.0

%PARAMS N^a Nz^b (N₊^c + N₋^c) (NS)^d Sz^e Pₐ^f cos(g*α) sin(h*α) Ny^(1-δ(0,h))
%Op;                Val;   Scl;   a;   b;   c;   d;   e;   f;   g;   h;  stg
μa;                 1.0;   0.0;   0;   1;   0;   0;   0;   0;   0;   0;   0
μa(3);            0.008;   0.0;   0;   1;   0;   0;   0;   0;   3;   0;   0
μb;                 1.0;   0.0;   0;   0;   1;   0;   0;   0;   0;   0;   0
μb(3);            0.008;   0.0;   0;   0;   1;   0;   0;   0;   3;   0;   0
μc(3);           -0.034;   0.0;   0;   0;   0;   0;   0;   0;   0;   3;   0
DN;    0.0020290470036744028;   0.0;   4;   0;   0;   0;   0;   0;   0;   0;   1
DNK;     0.12905102464425605;   0.0;   2;   2;   0;   0;   0;   0;   0;   0;   1
DK;       -1.016986627747583;   0.0;   0;   4;   0;   0;   0;   0;   0;   0;   1
d1;    0.0019484171306740406;   0.0;   2;   0;   2;   0;   0;   0;   0;   0;   1
d2;     -0.00250444477652152;   0.0;   0;   0;   4;   0;   0;   0;   0;   0;   1

