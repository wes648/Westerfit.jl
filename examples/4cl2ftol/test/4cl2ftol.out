4cl2ftol   @   2024-03-11T16:05:32.731

Control Parameters
mcalc = 5
νmax = 23.0
maxiter = 60
overwrite = true
Irrep = Ir
λlm0 = 0.0001
apology = true
Jmax = 16.5
S = 1.5
assign = RAM36
turducken = 3
REJECT = 10.0
TK = 25.0
RUNmode =  F
vtmax = 0
NFOLD = 3
νmin = 4.0
INTthres = 1.0e-5

Initial Parameters
 BK;          2314.3671312962265;   1.0
 BN;           769.4802420470164;   1.0
 B⨦;          48.512255192871294;   1.0
 Dab;        -126.68112589973111;   1.0
 T⁰₀(ϵ);                    -0.0;   0.0
 T²₀(ϵ);                     0.0;   0.0
 T²₁(ϵ);                     0.0;   0.0
 T²₂(ϵ);                     0.0;   0.0
 T²₀(χ);      -69.21833687588034;   1.0
 T²₁(χ);                    -0.0;   0.0
 T²₂(χ);      0.9871639007786803;   1.0
 F;            161663.7125406618;   0.0
 -2ρF;        -5957.395962170719;   1.0
 V3/2;       3.360317266162564e6;   1.0
 η;                          0.0;   0.0

 DN;        -8.097205890036752e-6;   1.0
 V3N;         -0.3798687828662461;   1.0
 V3N;                        -1.0;   1.0

Number of lines = 312

Initial RMS = 0.005252156728343281 MHz
Initial λ = 5.252142935840571e-10

-------------------------------------

he RMS has stopped decreasing. Hopefully it is low.

After   0 iterations:

Final RMS = 0.0053 MHz
Final log₁₀(λ) = -9.2797


                         Parameter                   Uncertainty
 A;             3028.964050901354;            0.1767022120662423
 B;              866.504752432759;           0.08007489277440097
 C;             672.4557316612738;           0.08007489277440097
 Dab;         -126.68112589973111;            0.9496134332968943
 ϵzz;                         0.0;                           0.0
 ϵxx;                         0.0;                           0.0
 ϵxz;                        -0.0;                           0.0
 ϵyy;                        -0.0;                           0.0
 χzz;          -69.21833687588034;           0.05051640486483865
 χxz;                         0.0;                           0.0
 χxx-χyy;      2.4180478494032083;           0.03092685095360825
 F;            5.3925209999999995;                           0.0
 ρ;          0.018425272649458393;          2.174297193498811e-5
 V3;            224.1762376932487;           0.08170551688021212
 η;                           0.0;                           0.0
 DN;        -8.097205890036752e-6;          1.522355148572828e-7
 V3N;         -0.3798687828662461;          0.013091235258855926
 V3N;                        -1.0;                           0.0


Journal Formatted Values
 A;                 3.02896(18)e3
 B;                  866.505(8)e0
 C;                  672.456(8)e0
 Dab;               -126.68(95)e0
 ϵzz;               0.0e0 (Fixed)
 ϵxx;               0.0e0 (Fixed)
 ϵxz;              -0.0e0 (Fixed)
 ϵyy;              -0.0e0 (Fixed)
 χzz;               -69.218(51)e0
 χxz;               0.0e0 (Fixed)
 χxx-χyy;             2.418(31)e0
 F;             5.39252e0 (Fixed)
 ρ;                 18.425(22)e-3
 V3;                224.176(82)e0
 η;                 0.0e0 (Fixed)
 DN;                  -8.1(15)e-6
 V3N;               -380.0(13)e-3
 V3N;               -1.e0 (Fixed)


 Again, sorry about the name...


